code/modules/projectiles/projectile/magic.dm

```diff
@@ -114,156 +114,8 @@
-			var/path
-			if(prob(50))
-				var/beast = pick("carp","bear","mushroom","statue", "bat", "goat","killertomato", "spiderbase", "spiderhunter", "blobbernaut", "magicarp", "chaosmagicarp", "watcher", "goliath", "headcrab", "morph", "stickman", "stickdog", "lesserdragon", "gorilla")
-				switch(beast)
-					if("carp")
-						path = /mob/living/simple_animal/hostile/carp
-					if("bear")
-						path = /mob/living/simple_animal/hostile/bear
-					if("mushroom")
-						path = /mob/living/simple_animal/hostile/mushroom
-					if("statue")
-						path = /mob/living/simple_animal/hostile/statue
-					if("bat")
-						path = /mob/living/simple_animal/hostile/retaliate/bat
-					if("goat")
-						path = /mob/living/simple_animal/hostile/retaliate/goat
-					if("killertomato")
-						path = /mob/living/simple_animal/hostile/killertomato
-					if("spiderbase")
-						path = /mob/living/simple_animal/hostile/poison/giant_spider
-					if("spiderhunter")
-						path = /mob/living/simple_animal/hostile/poison/giant_spider/hunter
-					if("blobbernaut")
-						path = /mob/living/simple_animal/hostile/blob/blobbernaut/independent
-					if("magicarp")
-						path = /mob/living/simple_animal/hostile/carp/ranged
-					if("chaosmagicarp")
-						path = /mob/living/simple_animal/hostile/carp/ranged/chaos
-					if("watcher")
-						path = /mob/living/simple_animal/hostile/asteroid/basilisk/watcher
-					if("goliath")
-						path = /mob/living/simple_animal/hostile/asteroid/goliath/beast
-					if("headcrab")
-						path = /mob/living/simple_animal/hostile/headcrab
-					if("morph")
-						path = /mob/living/simple_animal/hostile/morph
-					if("stickman")
-						path = /mob/living/simple_animal/hostile/stickman
-					if("stickdog")
-						path = /mob/living/simple_animal/hostile/stickman/dog
-					if("lesserdragon")
-						path = /mob/living/simple_animal/hostile/megafauna/dragon/lesser
-					if("gorilla")
-						path = /mob/living/simple_animal/hostile/gorilla
-			else
-				var/animal = pick("parrot","corgi","crab","pug","cat","mouse","chicken","cow","lizard","chick","fox","butterfly","cak")
-				switch(animal)
-					if("parrot")
-						path = /mob/living/simple_animal/parrot
-					if("corgi")
-						path = /mob/living/simple_animal/pet/dog/corgi
-					if("crab")
-						path = /mob/living/simple_animal/crab
-					if("pug")
-						path = /mob/living/simple_animal/pet/dog/pug
-					if("cat")
-						path = /mob/living/simple_animal/pet/cat
-					if("mouse")
-						path = /mob/living/simple_animal/mouse
-					if("chicken")
-						path = /mob/living/simple_animal/chicken
-					if("cow")
-						path = /mob/living/simple_animal/cow
-					if("lizard")
-						path = /mob/living/simple_animal/hostile/lizard
-					if("fox")
-						path = /mob/living/simple_animal/pet/fox
-					if("butterfly")
-						path = /mob/living/simple_animal/butterfly
-					if("cak")
-						path = /mob/living/simple_animal/pet/cat/cak
-					if("chick")
-						path = /mob/living/simple_animal/chick
```

Comment:
remind me to report whoever made this code, ill make it look better if this pr gets closed